torch/_torch_docs.py

```diff
@@ -3995,6 +3995,35 @@ def merge_dicts(*dicts):
     tensor(7.5107e-16, dtype=torch.float64)
 """.format(**common_args))
 
+add_docstr(torch.isin, r"""
+isin(elements, test_elements, *, assume_unique=False, invert=False) -> Tensor
```

Comment:
See comment in native_functions.yaml; this signature looks great, but the native_functions.yaml signatures allow `assume_unique` and `invert` to be specified positionally